curbs/README.md

```diff
@@ -503,7 +508,20 @@ A Location Reference is a JSON object with the following fields:
 | `side` | String | Optional | If the referenced linear feature is a roadway, the side of the roadway on which the Curb Zone may be found, when heading from the start to the end of the feature in its native orientation. Values are `left` and `right`. MUST be absent for features where `entire_roadway` is true. |
 
 [Top][toc]
-  
+
+## WZDx Reference
+
+A WZDx Reference object defines an external connection to a [Work Zone Data Exchange](https://github.com/usdot-jpo-ode/wzdx) data feed for a given curb place. This allows CDS users to reference public WZDx feeds that impact the area, and see full details in the agency data feed.
```

Comment:
> defines an external connection to a Work Zone Data Exchange data feed

I think it should be considered that as written, the WZDx Reference doesn't just define the connection to the feed, it describes a specific WZDx road event feature that is relevant to the curb zone.

Reply:
I made an update to incorporate your suggestions, let me know what you think.

Reply:
Looks good!